### PR TITLE
use css grid for amount buttons in update recurring

### PR DESCRIPTION
**Pull request by hejkal**
**NOTE: DO NOT discuss internal CommitChange information in your PR; this PR will be public.
Link back to the issue in the Tix repo when you need to do that.**

Updates the recurring donations editor to use css grid for the amount buttons, after making the same change for the widget in https://github.com/CommitChange/houdini/pull/941

### CODE REVIEW

File: client/js/recurring_donations/edit/amount-step.es6 [hunk 1]
```diff
@@ -69,9 +69,8 @@ function chooseNewDonationAmount() {
 // All the buttons and the custom input for the amounts to select
 function amountFields(state) {
     if(state.params$().single_amount) return ''
-    return h('div.u-inline.fieldsetLayout--three--evenPadding', [
-        h('span',
-            R.map(
+    return h('div.fieldset-grid', [
```
Comment (wwahammy) [inline client/js/recurring_donations/edit/amount-step.es6:72]: can we remove the wrapping array here?

For example, is there a reason that 


```javascript
[
   ...R.map(...., someArray)
]
```

can't be simplified to `R.map(...., someArray)`?

**Review by wwahammy — COMMENTED** [review]
Looks good, I just have one question

File: client/js/recurring_donations/edit/amount-step.es6 (hunk 1 above)
Comment (hejkal) [reply client/js/recurring_donations/edit/amount-step.es6:72]: Basically there are other things after the mapped items within the same array, so with the new CSS it's going from 
```js
[
   span with R.map(...., someArray) as its children array,
   custom amount,
   next button
]
```
to
```js
[
    ...R.map(...., someArray),
	custom amount,
	next button
]
```
putting the numbers and the other items all within one grid parent

Comment (wwahammy) [reply client/js/recurring_donations/edit/amount-step.es6:72]: Oops, I missed those. Thanks!

**Review by wwahammy — APPROVED** [review]

**Pull request merged**